com.ghostery.browser.yml

```diff
@@ -0,0 +1,102 @@
+---
+app-id: com.ghostery.browser
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: "."
+command: com.ghostery.browser
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--socket=pcsc"
+- "--socket=pulseaudio"
+- "--socket=cups"
+- "--env=GTK_PATH=/app/lib/gtkmodules"
+- "--persist=.ghostery browser"
+- "--filesystem=xdg-download:rw"
+- "--filesystem=xdg-run/pipewire-0"
+- "--device=dri"
+- "--talk-name=org.freedesktop.FileManager1"
+- "--system-talk-name=org.freedesktop.NetworkManager"
+- "--talk-name=org.a11y.Bus"
+- "--talk-name=org.gnome.SessionManager"
+- "--talk-name=org.freedesktop.ScreenSaver"
+- "--talk-name=org.gtk.vfs.*"
+- "--talk-name=org.freedesktop.Notifications"
+- "--env=MOZ_ENABLE_WAYLAND=1"
+- "--env=MOZ_USE_XINPUT2=1"
+- "--own-name=com.ghostery.browser.*"
+- "--own-name=org.mpris.MediaPlayer2.firefox.*"
+modules:
+- shared-modules/dbus-glib/dbus-glib-0.110.json
+- name: ghostery_browser
+  buildsystem: simple
+  build-commands:
+  - mkdir -p /app/lib
+  - mv ghostery_app/Ghostery /app/lib/ghostery
+  - install -D -m644 /app/lib/ghostery/browser/chrome/icons/default/default128.png
+    /app/share/icons/hicolor/128x128/apps/com.ghostery.browser.png
+  - install -D -m644 /app/lib/ghostery/browser/chrome/icons/default/default128.png
+    /app/usr/share/icons/hicolor/128x128/apps/com.ghostery.browser.png
```

Comment:
why do you install an icon in `/app/usr` ?

Reply:
It looks like another misunderstanding. I removed it.